engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -686,6 +686,89 @@ paths:
           description: At least one subsystem does not have an "ok" status
           schema:
             $ref: '#/definitions/StatusResponse'
+  '/api/ga4gh/wes/v1/service-info':
+    get:
+      tags:
+        - Workflow Execution Service - ALPHA
+      summary: Get service info
+      description: >-
+        May include information related (but not limited to) the workflow
+        descriptor formats, versions supported, the WES API versions supported,
+        and information about general service availability.
+      operationId: GetServiceInfo
+      parameters: []
+      responses:
+        '200':
+          description: Successful Request
+          schema:
+            $ref: '#/definitions/ServiceInfo'
```

Comment:
From a quick search, it looks like `openapi.yaml` files from version 3 onwards can [refer to external definitions](https://swagger.io/docs/specification/using-ref/). Perhaps worthy of a tech talk but I would advocate for referring to an original spec yaml rather than copy/pasting the reference definitions again here.

Reply:
Ah, never mind, this is a swagger 2.0 file 😢

Reply:
Follow up TOL: we could have probably replaced this entire API documentation with a ref that just said "we're implementing the API specified there. Kind of sad that we can't do that!